### ISSUE TITLE
Not possible to install without administrator rights (but usable when copied from other machine)

### ISSUE THREAD

**Issue by fcturner** [owncloud/client#3011]
I always use the client (Windows) at work. I am allowed to use such software, if it is free and does not need to be installed. And that is my point; it's not possible to install the client without admin rights. However, copying a installdirectory from another machine *does* work. 
So with every release, I install the client on my Windows computer at home, and just upload the Program Files\ownCloud content to my cloud and download it on my computer at work. It works perfectly (aside from the shell integration with Windows context menu).

Can you make it kind of portable? Dropbox did this too; when you don't have admin rights, not every component will be installed (like shell integration). But the most important part is, that it is still usable for end users.

**Comment by danimo** [comment]
Depends on https://github.com/owncloud/client/issues/860.
Requires research: How to get explorer plugins working in user installation?

**Comment by naderi** [comment]
I have just created a portable version of ownCloud Client with [yaP](http://rolandtoth.hu/yaP/) and pack it with [UPX](http://upx.sourceforge.net/). You can download it [here](https://mega.nz/#!88ojlC4T!CUQ_zxZQD_jeOJ4TLyf6i3Qlr8H2DypAZ9zpdvcD840)

**Comment by fcturner** [comment]
@naderi That's great for now. But it doesn't spare time, since it still requires extra actions.

I see that this issue is almost a year old now. @danimo You tell us it depends on #860. Well, that issue is being pushed to the future ever since 2013, and only changes milestones. Exactly the same as with this issue.. If I would constantly postpone my deadlines at work, my boss wouldn't be very happy :smile:

**Comment by aronovgj** [comment]
Making it portable is pretty easy. Just unpack the installer with 7zip or something similar. Just tried it, works.

I found this looking for a portable version for Linux. Couldn't find anything...

**Comment by tflidd** [comment]
@naderi put it in his github-repo: https://github.com/naderi/owncloud-client-portable/

**Comment by danimo** [comment]
Note: The "portable" version lacks the Windows Explorer integration. Also, this is not being tested by the core team.

**Comment by fcturner** [comment]
Any news?

**Comment by hodyroff** [comment]
It seems that the idea of a non-admin version would be to give up on some features. Generally this is not asked very often, but if somebody wants to step up and work on it, you will be welcome.

**Comment by Sigmun** [comment]
This issue still keep me from using owncloud at work... Very annoying!

**Comment by guruz** [comment]
@dschmidt Is this easier now with the new MSI installer?

**Comment by hodyroff** [comment]
@Sigmun Why does the portable version not work for you?
@guruz  @dschmidt No, the Windows File Explorer pieces need administrator rights.
If the portable version works, I wonder whats missing exactly.

**Comment by guruz** [comment]
@naderi Do you want to update your portable version to 2.5.4? Or contribute your build scripts for it, maybe we can integrate it with the ownCloud build servers?

@hodyroff I guess https://github.com/naderi/owncloud-client-portable/ does not look official enough.

**Comment by guruz** [comment]
@dschmidt Would one copy over https://github.com/owncloud/craft-blueprints-owncloud/blob/dep-tracking/owncloud/owncloud-client/owncloud-client.py#L34 to a owncloud-client-portable and go from there?

**Comment by dschmidt** [comment]
Well, we have sort of a portable package as an interim build step. It needs testing and everything and possibly stripping of unneeded assets, but it shouldn't be too much work

**Comment by michaelstingl** [comment]
> Well, we have sort of a portable package as an interim build step.

@Sigmun Would you be interested in testing @dschmidt 's "portable package" thingy?

**Comment by Sigmun** [comment]
> @Sigmun Would you be interested in testing @dschmidt 's "portable package" thingy?

Yes I can test it...

**Comment by michaelstingl** [comment]
> Well, we have sort of a portable package as an interim build step.

@TheOneRing Could you get us the portable thing from the build chain?

**Comment by sandyscott** [comment]
The portable owncloud client project appears to have been abandonned.

I think it would definitely be preferable for the official installer to have a fallback mode where it runs without admin privileges, though I can see that it involves more changes to the install process that just the installation folder (like explorer integration as mentioned above). It'll be interesting to see how the massive overhaul to Explorer in Windows 11 changes things in this regard.

**Comment by DeepDiver1975** [comment]
out of scope
